### ISSUE TITLE
Events datepicker should update end date automatically when start date is changed

### ISSUE THREAD

**Issue by danfox01** [baltimorecounty/BCPL-assets#569]
I am 99% sure the datepicker used to work differently in order to reduce incidence of user error. I'm not sure when this changed.

**Current behavior:**
Changing the start date doesn't have any bearing on the end date. So in most cases, selecting a start date out will immediately cause the page to yell at the user because their end date is no longer invalid.

**Expected behavior:**
Changing the start date automatically selects the end date as the next day. E.g., entering Sept. 22 automatically changes the end date to Sept. 23. This makes it a lot more difficult for a user to enter an invalid date range.

**Comment by danfox01** [comment]
@tmccoy529 @martypowell This issue has changed a little bit. The bug fix we did with the date picker race condition (#560) also eliminated part of this issue. Still a good UI improvement so I updated and retagged as an enhancement.

**Comment by tmccoy529** [comment]
@danfox01 @martypowell yeah Ive played around with this and got a lots of unexpected behaviors so Im glad to see that only when the start date is greater is when we will change the end date. I believe that will remove those weird issues I was seeing.

**Comment by tmccoy529** [comment]
@martypowell @danfox01 I have put a pull request out for this. Once we work through that it will be ready for testing.

**Comment by tmccoy529** [comment]
@danfox01 this is ready for testing

**Comment by danfox01** [comment]
@tmccoy529 took a quick look and the datepicker functionality seems spot on to me. I will pass to @jdomasky for thorough testing.

@jdomasky, please run this one through the events page general test plan to make sure all the existing functionality still works as expected. Please also put the new date behavior through the paces.

**Comment by jdomasky** [comment]
@danfox01 OK, I'll test this today.

**Comment by jdomasky** [comment]
@tmccoy529 @martypowell @danfox01 Correction: I'll test this issue next week.

**Comment by jdomasky** [comment]
The Events interface in dev passed the Events Test Plan completely.
In addition, I thoroughly tested the new date behavior and it works well in all cases. I also has a look via BrowserStack iPhone Safari.
